roles/ansible-keylime-tpm20/tasks/git-repos.yml

```diff
@@ -0,0 +1,35 @@
+- name: Clone python-keylime
+  git:
+    repo: https://github.com/keylime/python-keylime
+    dest: /root/python-keylime
+
+- name: Clone tpm2 software stack
+  git:
+    repo: https://github.com/tpm2-software/tpm2-tss
+    dest: /root/tpm2-tss
+    version: a0d6f7d
+
+- name: Clone tpm2 tools
+  git:
+      repo: https://github.com/tpm2-software/tpm2-tools
```

Comment:
For now we need to use keylime/tpm2-tools as there is an issue with the ESAPI that is not fixed yet:

This is the following issue:

https://github.com/keylime/python-keylime/issues/59

I would recommend to repoint this to 

https://github.com/keylime/tpm2-tools

And I will make an issue to make sure when #59 is fixed upstream, we update this role!